network/transport/grpc/authenticator.go

```diff
@@ -0,0 +1,63 @@
+package grpc
+
+import (
+	"fmt"
+	"github.com/nuts-foundation/go-did/did"
+	"github.com/nuts-foundation/nuts-node/network/log"
+	"github.com/nuts-foundation/nuts-node/network/transport"
+	"github.com/nuts-foundation/nuts-node/vdr/doc"
+	"google.golang.org/grpc/credentials"
+	grpcPeer "google.golang.org/grpc/peer"
+	"net/url"
+)
+
+const nutsCommServiceType = "NutsComm"
```

Comment:
you might want to place it in a more general place since it'll be used for service discovery as well?

Reply:
Then we'll see then where it should fit in